dev/sbt-plugin/src/sbt-test/sbt-plugin/projections-happy-path/hello/src/main/resources/application.conf

```diff
@@ -0,0 +1,20 @@
+play.modules.enabled += com.example.hello.impl.HelloModule
+
+lagom.projection.auto-start.enabled = false
+
+## Few hacks to speed up the test
+## speed up the ensure-active-interval to distribute workers faster
+lagom.persistence.cluster.distribution.ensure-active-interval = 2s
+## reduce eventual consistency
+cassandra-query-journal.eventual-consistency-delay = 2s
+## set first-time-bucket to speed up unit tests (starting with no offset is faster with this.)
+cassandra-query-journal {
+  first-time-bucket = "20190903T00:00"
+}
```

Comment:
These few settings above made a huge difference in times to run the test (unit test and dev mode).

Reply:
The first-time-bucket value made a huge difference: https://github.com/lagom/lagom/issues/2236